File: eAirFlow.Model/Models/Payment.cs
using System;
using System.Collections.Generic;

namespace eAirFlow.Model.Models
{

    public class Payment
    {
        public int PaymentId { get; set; }

        public int? UserId { get; set; }

        public decimal? Amount { get; set; }

        public string? PaymentMethod { get; set; }

        public DateTime? TransactionDate { get; set; }

        public string? TransactionReference { get; set; }

    }
}

File: UI/eairflow_desktop/lib/models/payment.dart
class Payment {
  int? paymentId;
  int? userId;
  double? amount;
  String? paymentMethod;
  DateTime? transactionDate;
  String? transactionReference;

  Payment({
    this.paymentId,
    this.userId,
    this.amount,
    this.paymentMethod,
    this.transactionDate,
    this.transactionReference,
  });

  factory Payment.fromJson(Map<String, dynamic> json) {
    return Payment(
      paymentId: json['paymentId'],
      userId: json['userId'],
      amount: json['amount'] != null 
          ? (json['amount'] as num).toDouble()
          : null,
      paymentMethod: json['paymentMethod'],
      transactionDate: json['transactionDate'] != null
          ? DateTime.parse(json['transactionDate'])
          : null,
      transactionReference: json['transactionReference'],
    );
  }
}
